repository/snowflake.go

```diff
@@ -237,6 +244,9 @@ func (s *Snowflake) GetMaxValue(ctx context.Context, table, orderingColumn strin
 	if err != nil {
 		return nil, fmt.Errorf("query get max value: %w", err)
 	}
+	if rows.Err() != nil {
+		return nil, fmt.Errorf("query get max value: %w", err)
+	}
```

Comment:
We actually need to move these checks after calling `rows.Next()`, this error tells you if/what went wrong when iterating through the result set (see [Errors From Iterating Resultsets](http://go-database-sql.org/errors.html#errors-from-iterating-resultsets)).